diff -up fltk-1.3.0r9619/src/Fl_x.cxx.expose fltk-1.3.0r9619/src/Fl_x.cxx
--- fltk-1.3.0r9619/src/Fl_x.cxx.expose	2012-07-05 09:49:55.466925868 +0200
+++ fltk-1.3.0r9619/src/Fl_x.cxx	2012-07-05 09:50:24.159428363 +0200
@@ -1838,14 +1838,13 @@ int fl_handle(const XEvent& thisevent)
 void Fl_Window::resize(int X,int Y,int W,int H) {
   int is_a_move = (X != x() || Y != y());
   int is_a_resize = (W != w() || H != h());
-  int is_a_enlarge = (W > w() || H > h());
   int resize_from_program = (this != resize_bug_fix);
   if (!resize_from_program) resize_bug_fix = 0;
   if (is_a_move && resize_from_program) set_flag(FORCE_POSITION);
   else if (!is_a_resize && !is_a_move) return;
   if (is_a_resize) {
     Fl_Group::resize(X,Y,W,H);
-    if (shown()) {redraw(); if(is_a_enlarge) i->wait_for_expose = 1;}
+    if (shown()) redraw();
   } else {
     x(X); y(Y);
   }
